### ISSUE TITLE
Rascal: org.rascalmpl.interpreter.staticErrors.UnexpectedKeywordArgumentType from MetricListExecutor 

### ISSUE THREAD

**Issue by jimmysyl** [ossmeter/ossmeter#56]
The following exception is thrown for the following projects: OSSMETER, EMF.

The stack trace is:

``` Java
org.rascalmpl.interpreter.staticErrors.UnexpectedKeywordArgumentType: Expected map[str, int], but got
map[loc, int] for keyword parameter prev [Advice](http://tutor.rascal
mpl.org/Errors/Static/UnexpectedKeywordArgumentType/UnexpectedKeywordArgumentType.html) at
org.rascalmpl.interpreter.result.AbstractFunction.bindKeywordArgs(AbstractFunction.java:543) at
org.rascalmpl.interpreter.result.RascalFunction.call(RascalFunction.java:267) at
org.ossmeter.metricprovider.rascal.RascalMetricProvider.compute(RascalMetricProvider.java:302) at
org.ossmeter.metricprovider.rascal.RascalMetricProvider.measure(RascalMetricProvider.java:206) at
org.ossmeter.metricprovider.rascal.RascalMetricProvider.measure(RascalMetricProvider.java:1) at
org.ossmeter.platform.osgi.executors.MetricListExecutor.run(MetricListExecutor.java:130) at
java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145) at
java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615) at
java.lang.Thread.run(Thread.java:745)
```

**Comment by basten** [comment]
This is probably a duplicate of #39

**Comment by basten** [comment]
@jimmysyl Do you know the metric that triggered this error?

**Comment by nmatra** [comment]
@basten MetricListExecutor: rascal.generic.churn.coreCommittersChurn

**Comment by basten** [comment]
Thanks @nmatra. I couldn't find anything strange about that metric. However, on Nov 24 I changed the data type of the metric from map[loc, int] to map[str, int]. Could it be you're using an older database that contains data of the old type?

**Comment by basten** [comment]
Closed b/c #39 is closed
